test_enclave/build.rs

```diff
@@ -149,7 +178,10 @@ where
 ///
```

Comment:
Ah I see what you mean, I should pass `keyfile.is_some()` rather than the whole `keyfile` since its value isn't used. Fair enough.

I am hesitant to link the PCL library when it's not used... just because I haven't dug into it very deeply so I don't really know whether it would cause some issue down the line.